DigitalLearningSolutions.Web/Helpers/RegistrationEmailValidator.cs

```diff
@@ -0,0 +1,71 @@
+﻿namespace DigitalLearningSolutions.Web.Helpers
+{
+    using DigitalLearningSolutions.Data.Services;
+    using DigitalLearningSolutions.Web.Extensions;
+    using DigitalLearningSolutions.Web.ViewModels.Register;
+    using Microsoft.AspNetCore.Mvc.ModelBinding;
+
+    public static class RegistrationEmailValidator

```

Comment:
This is going to cause a merge conflict with the bug fix PR for 886 https://github.com/TechnologyEnhancedLearning/DLSV2/pull/1163.
I expect that will get merged first, so this will need to make sure it has the updated logic/messages.